VMBackup/main/common.py

```diff
@@ -42,6 +42,11 @@ class CommonVariables:
     statusBlobUploadError = 'statusBlobUploadError'
     TempStatusFileName = 'tempStatusFile.status'
     onlyLocalFilesystems = 'onlyLocalFilesystems'
+    isSnapshotTTLEnabled = 'isSnapshotTTLEnabled'
+    useMCCFForLAD = ' UseMCCFForLAD'
+    useMCCFToFetchDSASForAllDisks = 'UseMCCFToFetchDSASForAllDisks'
+    settings = 'settings'
```

Comment:
what is the use of settings constant?

Reply:
using it here,
 https://github.com/Azure/azure-linux-extensions/pull/1712/files#:~:text=if(-,CommonVariables.settings,-in%20self. 
 we can avoid it as well